docs/markdown/Builtin-options.md

```diff
@@ -113,6 +115,21 @@ the two-way mapping:
 
 All other combinations of `debug` and `optimization` set `buildtype` to `'custom'`.
 
+#### Details for `warning_level`
+
+Exact flags per warning level is compiler specific, but there is an approximative
+table for most common compilers.
+
+| Warning level | GCC/Clang                | MSVC  |
+| ------------- | ---                      | ----  |
+| 0             |                          |       |
+| 1             | -Wall                    | /W2   |
+| 2             | -Wall -Wextra            | /W3   |
+| 3             | -Wall -Wextra -Wpedantic | /W4   |
+| everything    | -Weverything             | /Wall |
```

Comment:
Perhaps it would be useful to wrap the compiler args in backticks.

Reply:
It renders well as is, most tables in that page does not use backticks.